backend/src/models/user.js

```diff
@@ -0,0 +1,9 @@
+import crypto from 'crypto'
+
+export class User {
```

Comment:
This model does not match the current target schema!
please Update it.
And also add comments for the class.